src/ResourceManagement/HDInsight/HDInsight/Properties/AssemblyInfo.cs

```diff
@@ -33,7 +33,7 @@
 // by using the '*' as shown below:
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Note: if you want the full version in the UA, you can change the AssemblyVersion here